doc/tools/coqrst/coqdomain.py

```diff
@@ -1158,7 +1158,8 @@ def GrammarProductionRole(typ, rawtext, text, lineno, inliner, options={}, conte
 
 def GlossaryDefRole(typ, rawtext, text, lineno, inliner, options={}, content=[]):
     """Marks the definition of a glossary term inline in the text.  Matching :term:`XXX`
-    constructs will link to it.  The term will also appear in the Glossary Index.
+    constructs will link to it.  Use the form :gdef:`text<term>` to display "text"
+    for the definition of "term".  The term will also appear in the Glossary Index.
```

Comment:
I think the doc should make it clear that this should only be used to put a capital letter or a plural form but not something different.

Reply:
Usually in reST there's a space before the `<`, and `text` is called `title` (though here `text is good too)

Reply:
> I think the doc should make it clear that this should only be used to put a capital letter or a plural form but not something different.

I added wording suggesting those uses.  I'm reluctant to make it imperative--I have trouble imagining that others would commonly be tempted to misuse the feature.  Is that OK?

> Usually in reST there's a space before the `<`

I wasn't aware that a space was allowed until, I think, yesterday.  My impression is that most of our uses of `< ... >` don't have the space.  It is more readable with the space, we should prefer having the space.

Reply:
You haven't pushed yet, so I cannot comment on your wording. But sure, no need to make it imperative, as long as the expected use case is clearly explained.

Reply:
How's this?

```
    Marks the definition of a glossary term inline in the text.  Matching :term:`XXX`
    constructs will link to it.  Use the form :gdef:`text <term>` to display "text"
    for the definition of "term", such as when "term" must be capitalized or plural
    for grammatical reasons.  The term will also appear in the Glossary Index.
```

Reply:
LGTM